test/Garnet.test/RespSetTest.cs

```diff
@@ -273,8 +273,86 @@ public void CanDoSScanWithCursor()
             Assert.AreEqual(l, $"value:{setEntries.Length - 1}");
         }
 
+        [Test]
+        public void CanDoSdiff()
+        {
+            using var redis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            var db = redis.GetDatabase(0);
+
+            var key1 = "key1";
+            var key1Value = new RedisValue[] { "a", "b", "c", "d" };
+
+            var key2 = "kye2";
+            var key2Value = new RedisValue[] { "c" };
+
+            _ = db.SetAdd(key1, key1Value);
```

Comment:
Would probably be good to assert that SetAdd was successful (i.e. Assert.AreEqual(1, result));